devicetypes/tommysqueak/danfoss-living-connect-radiator-thermostat.src/danfoss-living-connect-radiator-thermostat.groovy

```diff
@@ -44,65 +45,73 @@ metadata {
+				attributeState "VALUE_DOWN", action: "tempDown"
+			}
+
+			tileAttribute("device.battery", key: "SECONDARY_CONTROL") {
+				attributeState "default", label:'${currentValue}%', unit:"%", icon:"st.nest.empty" // TODO: update battery icon
 			}
 
 			tileAttribute("device.thermostatOperatingState", key: "OPERATING_STATE") {
-				attributeState("default", label:'${currentValue}')
-				attributeState("heating", label:"heating", backgroundColor:"#ffa81e", icon:"st.thermostat.heat")
-				attributeState("idle", label:"idle", backgroundColor:"#1e9cbb", icon:"st.thermostat.heating-cooling-off")
+				attributeState "default", label:'${currentValue}'
+				attributeState "heating", label:"heating", backgroundColor:"#ffa81e", icon:"st.thermostat.heat"
+				attributeState "idle", label:"idle", backgroundColor:"#1e9cbb", icon:"st.thermostat.heating-cooling-off"
 			}
 
 			tileAttribute("device.thermostatMode", key: "THERMOSTAT_MODE") {
-				attributeState("default", label:'${currentValue}')
-				attributeState("heat", label:"heat", backgroundColor:"#ffa81e", icon:"st.thermostat.heat")
-				attributeState("off", label:"off", backgroundColor:"#1e9cbb", icon:"st.thermostat.heating-cooling-off")
+				attributeState "default", label:'${currentValue}'
+				attributeState "heat", label:"heat", backgroundColor:"#ffa81e", icon:"st.thermostat.heat"
+				attributeState "off", label:"off", backgroundColor:"#1e9cbb", icon:"st.thermostat.heating-cooling-off"
 			}
 
 			tileAttribute("device.heatingSetpoint", key: "HEATING_SETPOINT") {
-					attributeState("default", label:'${currentValue}', unit:"°C")
+				attributeState "default", label:'${currentValue}', unit:"°C"
 			}
 		}
 
-		valueTile("battery", "device.battery", inactiveLabel: false, height: 2, width: 3, decoration: "flat") {
-			state "battery", label:'${currentValue}% battery', unit:""
+		standardTile("switcher", "device.switch", height: 2, width: 2, decoration: "flat") {
+			state "off", action:"on", label: "off", icon: "st.thermostat.heating-cooling-off", backgroundColor:"#ffffff"
+			state "on", action:"off", label: "on", icon: "st.thermostat.heat", backgroundColor:"#79b821"
 		}
 
-		standardTile("switcher", "device.switch", height: 2, width: 2, decoration: "flat") {
-			state("off", action:"on", label: "off", icon: "st.thermostat.cool", backgroundColor:"#ffffff")
-			state("on", action:"off", label: "on", icon: "st.thermostat.heat", backgroundColor:"#ffa81e")
+		standardTile("thermostatMode", "device.thermostatMode", height: 2, width: 2, decoration: "flat") {
+			state "off", action:"heat", icon: "st.thermostat.heating-cooling-off"
+			state "heat", action:"off", icon: "st.thermostat.heat"
+		}
+
+		valueTile("nextHeatingSetpoint", "device.nextHeatingSetpoint", height: 2, width: 2, decoration: "flat") {
+			state "default", label:'${currentValue}°', unit:"dC"
+		}
+
+		valueTile("heatingSetpoint", "device.heatingSetpoint", height: 2, width: 2, decoration: "flat") {
```

Comment:
Is this and the `nextHeatingSetpoint` tiles needed, if they're shown via the `details([])` line?